src/lib/numbers.ts

```diff
@@ -16,6 +16,30 @@ export function expandDecimals(n: BigNumberish, decimals: number): BigNumber {
   return bigNumberify(n).mul(bigNumberify(10).pow(decimals));
 }
 
+const MAX_EXCEEDING_THRESHOLD = "1000000000";
+const MIN_EXCEEDING_THRESHOLD = "0.01";
+
+function checkPriceThresholdExceeding(value: BigNumber, tokenDecimals: number) {
```

Comment:
not sure about the naming - “check” looks like something that returns boolean

Maybe we could call it like “getLimitedDisplay” or something else